src/main/java/ch/njol/skript/conditions/CondServerTickState.java

```diff
@@ -34,6 +34,8 @@
 import org.bukkit.event.Event;
 import org.eclipse.jdt.annotation.Nullable;
 
+import java.util.Locale;
+
 @Name("Server Tick State")
 @Description("Represents the ticking state of the server, for example, if the server is frozen, or running normally.")
 @Examples({"if server's tick state is currently frozen:", "if server tick state is normal:"})
```

Comment:
```suggestion
@Examples({
	"if server's tick state is currently frozen:",
	"if server tick state is normal:"
})
```
do this for the other examples too